### PR TITLE
Enhanced docblock for tribe_get_view_permalink function

### PR DESCRIPTION
**Pull request by Sjaure**
- Added a detailed description of the function's parameters and return value.
- Included information about the applied filters and their parameters.

### 🎫 Ticket

[TICKET_ID]

### ✔️ Checklist
- [ ] Changelog entry in the `readme.txt` file.
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [ ] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

### CODE REVIEW

File: src/functions/template-tags/link.php [hunk 1]
```diff
@@ -145,9 +145,7 @@ function tribe_get_events_link( $context = 'href' ) {
  *
  * @param string        $slug      The slug of the view for which the permalink is being generated.
  * @param bool|int|null $term      Optional. The term associated with the view. Default is null.
- * @param string        $permalink The generated permalink.
- * 
- * @return string The generated permalink for the view.
+ * @param string        $permalink The generated permalink for the view.
```
Comment (bordoni) [inline src/functions/template-tags/link.php:148]: It doesnt seem like this method has `$permalink` so we should address that.

**Review by bordoni — CHANGES_REQUESTED** [review]

File: src/functions/template-tags/link.php [hunk 2]
```diff
@@ -136,11 +136,16 @@ function tribe_get_events_link( $context = 'href' ) {
...
- * @param bool|int|null $term
+ * @since 5.7.0
+ * 
+ * @hook tribe_get_view_permalink        Filters the overall view permalink.
+ * @hook tribe_get_{slug}_view_permalink Filters the specific view permalink.
  *
- * @return string $permalink
+ * @param string        $slug      The slug of the view for which the permalink is being generated.
+ * @param bool|int|null $term      Optional. The term associated with the view. Default is null.
+ * @param string        $permalink The generated permalink for the view.
```
Comment (Camwyn) [inline src/functions/template-tags/link.php:148]: I think G meant the return is missing - it got changed to a param by accident:
```suggestion
 * 
 * @return string        $permalink The generated permalink for the view.
```

**Review by Camwyn — CHANGES_REQUESTED** [review]

File: src/functions/template-tags/link.php (hunk 2 above)
Comment (Camwyn) [reply src/functions/template-tags/link.php:148]: Check the spacing on that line I suggested - it might be off 😉

Comment (Sjaure) [reply src/functions/template-tags/link.php:148]: Aha! That's right. Thanks

**Review by Camwyn — APPROVED** [review]

**Review by bordoni — APPROVED** [review]

**Pull request merged**